R/derive_param_confirmed_resp.R

```diff
@@ -0,0 +1,382 @@
+#' Derive Confirmed Response Parameter
+#'
+#' Derive confirmed response parameter
```

Comment:
i commented on the other function that this looks like a repeat of row 1 but now i'm thinking maybe this must be intentional to get the title to show or something like that??